tests/cpp_api_tests/rocAL_audio_unittests/rocal_audio_unittest.py

```diff
@@ -72,7 +72,8 @@ def main():
         sys.exit()
 
     sys.dont_write_bytecode = True
-    input_file_path = rocal_data_path + "/audio/wav"
+    input_file_path = rocal_data_path + "/audio"
+    input_file_list_path = rocal_data_path + "/audio/wav_file_list.txt"
```

Comment:
I would prefer not to pass this file list path or make the changes here for file list